widget/edit.html

```diff
@@ -0,0 +1,171 @@
+                            <input name="layer" type="text" data-ng-model="item.title" class="form-control" required />
+                            <div
+                                data-ng-class="{ 'has-error': editFunnelWidgetForm.moduleValue.$invalid && editFunnelWidgetForm.moduleValue.$touched }">
+                                <label for="moduleValue" class="control-label">Data Source <span
+                                        class="text-danger">*</span>
+                                </label>
+                                <select name="moduleValue" id="moduleValue{{$index}}" class="form-control"
+                                    data-ng-options="module.type as module.name for module in modules"
+                                    data-ng-model="item.value" data-ng-change="loadAttributes($index)" required>
+                                    <option value="">Select an Option</option>
+                                </select>
+                            </div>
+                            <div class="margin-top-md margin-bottom-md" data-ng-if="item.fields !== undefined">
+                                <label class="control-label">Filter Criteria <span class="text-danger">*</span>
+                                </label>
+                                <div data-cs-conditional data-ng-if="item.fieldsArray.length > 0"
+                                    data-fields="item.fields" data-reset-field="item.fields" data-mode="'queryFilters'"
+                                    data-ng-model="item.query" data-parent-form="editFunnelWidgetForm"
+                                    data-enable-expression="(page==='dashboard' || page==='reporting')"
+                                    data-show-uuid="true" data-form-name="'editFunnelWidgetForm'"></div>
+                            </div>
+                            <button type="button" data-ng-click="removeLayer($index)" id="fsr-module-{{ $index }}"
+                                class="btn-sm btn btn-link btn-conditional-delete" style="position: absolute;"
+                                data-ng-if="$index !== 0">
+                                <span class="icon icon-close text-danger"></span>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div data-ng-if="config.funnelModuleType !== undefined">
+                    <div class="row margin-left-0 margin-top-lg margin-bottom-lg" style="padding-left: 5px;"
+                        data-ng-hide="processing || disabled">
+                        <button data-ng-click="addLayer()" type="button" class="btn-btn btn-sm btn-primary">+ &nbsp;Add
+                            Layer</button>
+                    </div>
+                </div>
+                <div class="custom-alert custom-alert-danger margin-bottom-xlg margin-top-md"
+                    data-ng-if="maxlayers === true">
+                    <span
```

Comment:
![Screenshot 2023-02-17 at 12 09 31 PM](https://user-images.githubusercontent.com/39477678/219568554-f122965c-e3f3-43b7-b132-665ef33bfaf3.png)


alert text not readable clearly, please use alert boxes which are same overall the application